pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "prometheus_client"
+version = "0.21.1"
```

Comment:
This also changed to match the most recent release which had a backport applied to it.